config/environment.js

```diff
@@ -20,6 +20,9 @@ module.exports = function(environment) {
     sentry: {
       environment,
     },
+    apollo: {
```

Comment:
are you using apollo? Don't use the URL pointing directly at client-api.

Reply:
I missed to delete these